helm/older-versions/SP.2.N.yaml

```diff
@@ -0,0 +1,63 @@
+repository:
```

Comment:
As discussed please remove this for now, but keep a copy as we may well need it soon, depending on the decision in OPSEXP-24

Reply:
yes, will add it to the OPSEXP-24 issue.